iowrappers/maps_client.go

```diff
@@ -62,3 +62,23 @@ func CreateLogger() error {
 
 	return nil
 }
+
+func (mapsClient *MapsClient) ReverseGeocoding(context context.Context, latitude, longitude float64) (GeocodeQuery, error) {
+	request := &maps.GeocodingRequest{
+		LatLng: &maps.LatLng{
+			Lat: latitude,
+			Lng: longitude,
+		},
+		// currently we only need country and city info
+		ResultType: []string{"country", "locality"},
```

Comment:
What is Locality? or is `city` called `locality` ?

Reply:
city called locality